### DISCUSSION TITLE
What is the `x-sveltekit-page: true` response header and where does it come from? (Vercel, sveltekit 1.10)

### DISCUSSION THREAD

**Discussion by manvankaliber** [sveltejs/kit#9405] [Q&A]
It was spotted on one of my broken ISR routes that I'm trying to debug. Does SvelteKit add it, or Vercel? And why is it added? The reason for me to ask is because I have this weird bug with one specific ISR route that for some reason always returns a 404, even when there is only a `return new response('hello')` in its GET handler... 

I'm think the router of Svelte Kit is broken, or maybe it is the Vercel adapter... or maye it is Vercels platform itself?...

I actually do not know who or what to blame yet... but I hope answers to this question will point me in the right direction :)

![image](https://user-images.githubusercontent.com/12383587/224487205-a7e06b5b-99e8-498d-8c73-508b0b88b896.png)

**Comment by wighawag** [comment]
I was wondering the same and found your question

Seems like it is coming from server rendered page when not served as chunks : https://github.com/sveltejs/kit/blob/81af6baf9b6f6e8f14212958eb393677db09320a/packages/kit/src/runtime/server/page/render.js#L390

**Comment by Fish1** [comment]
Same issue here...

This link is broken
https://yapms2.com/app/usa/governors/2024

This link works
https://yapms2.com/app/usa/senate/2024

We are using dynamic routes app/[country]/[map]/[year] and it is prerendered. We then parse those variables on the frontend to dynamically load different data.

![image](https://github.com/sveltejs/kit/assets/13600696/80b2ab9c-af6c-4648-806e-5d56db27da3a)

![image](https://github.com/sveltejs/kit/assets/13600696/4cc31256-59a1-46bd-9200-a40a84c51245)
